src/middlewared/middlewared/plugins/jail.py

```diff
@@ -683,22 +683,24 @@ def fstab(self, jail, options):
 
         return True
 
-    @accepts(Str("pool"))
+    @accepts(Str("pool", required=True))
```

Comment:
Did you try without required=True? It should be required regardless of that attribute, which only really makes sense in Dict.

Reply:
Yeah without it the method allowed nothing to be supplied at all. Perhaps a regression?

Reply:
11.3, without adding required it wouldn't throw the schema exception.